Troubleshooting.md

```diff
@@ -70,17 +78,17 @@ Once the collection barcode duplication issue is resolved, the manifest ETL will
 Aborting with error: Identifier found in set «samples-haarvi», not «samples»
 ```
 This error means that a sample from a separate study arm that we're not supposed to be ingesting was not properly marked as experimental (`_exp`), so it ended up in our pipeline.
-The appropriate avenue is to Slack someone in one of data-transfer channels.
-We can ask NWGC to re-send the same JSON bundle but with `_exp` designations on the affected samples.
-We should manually skip the bundle in `recieving.presence_absence` and wait for the updated JSON.
-Find the presence_absence_id/group number by looking for 
+##### Solution:
+Contact the appropriate data-transfer channel in Slack. Copy-paste the error message along with the presence_absence_id/group number into the message.
+Find the presence_absence_id/group number by looking through /var/log/syslog on the backoffice server for 
 ```
 Rolling back to savepoint presence_absence group {the group number}
 ````
-in /var/log/syslog.
 
-Manually mark the receiving record as skipped like this:
-````sql
+We can ask NWGC to re-send the same JSON bundle but with `_exp` designations on the affected samples.
+To manually skip the bundle in `recieving.presence_absence`:
```

Comment:
Can we add additional context here, something like:

If a presence absence bundle (results plate) is ever skipped as an interim measure, we must communicate to the lab on slack that the plate will need to be retransferred from Samplify to ID3C. Otherwise, any results from that plate will not be processed by ID3C nor made available to participants.